lib/utils/fs/writeFile.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const path = require('path');
+const YAML = require('js-yaml');
+const fse = require('./fse');
+
+function writeFile(filePath, conts) {
+  let contents = conts || '';
+
```

Comment:
A check if the `filePath` is present should be added here (and should throw an error)